### DISCUSSION TITLE
I'm having this issue with cloning repository since I did a system restore on my computer

### DISCUSSION THREAD

**Discussion by JakeJakobs** [community/community#89917] [Programming Help]
### Body

I usually get the repository by cloning the HTTPS with Powershell onto desktop, and it used to work pretty darn well for me until I did a system restore. 

Why did I do a system restore? I somehow managed to get a trojan virus when I was looking for 3rd-party tools on Reddit, found myself in a dead link and that's how I acquired the disgusting virus. The system restore got rid of the virus though (or at least I hope so, worst case it's still there incognito now and is causing more issues), and now I am unable to use GitHub due to, what I assume to be network issues or issues with GitHub itself. 

![0101githubproblem010101](https://github.com/community/community/assets/144908118/8533a3e7-3c76-486d-99cf-5383c498d164)

I can't download ZIP either, same issue happens, when I try to open with GitHub desktop, it does not open or load. So this issue is making it impossible for me to pull anything out of my repositories. 

(Note: I have a habit of deleting the repository file from documents when I encounter problems with the project on Unity, to effectively refresh by cloning the repository back onto desktop and into respective git document folder.) 

If anyone has seen issues similar to this or are aware of what could be happening, please help me out if you are up for it, I am not the smartest of people so I'll do my best to work with whatever help you offer me.

Thank you. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by jkoda-rsa** [comment]
The error messages you're seeing typically point to network-related issues during Git operations. Here are a few suggestions you can try to resolve these problems:

1. **Increase Git Buffer Size**: If the repository you're trying to clone is too large for the default Git buffer size, you can increase it with the following command:

   ```bash
   git config --global http.postBuffer 524288000

this will set the buffer size to 500MB, this might work for you. 

2. **Disable Git’s Compression Settings**: I am not entirely sure, but your problem could be related to compression of data during transfer, you can disable Git’s compression with this command:

   ```bash
   git config --global core.compression 0

If you are still experiencing the issue, you could try to clone it like this: 

   ```bash
   GIT_TRACE=1 GIT_CURL_VERBOSE=1 git clone <repository-url>
   ```


You can enable a more verbose output to gain more information.

Hope this helps!

Cheers,
Jenestin

**Reply by JakeJakobs** [reply]
Thank you for your response, it took me a little while to respond because I did a factory reset and it seemed to have fixed the issue, seemed to, but now I am encountering the same issue. So maybe it's not my hardware, maybe it's the connection to GitHub like you said, who knows.
The error is quite persistent, but doesn't always show up, almost as if tricking me into thinking its fixed when its actually not. 

Anyway I tried your suggested methods and this is what happened ...

---

When I enter first and second step this message comes up in red:
git : The term 'git' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ git config --global http.postBuffer 524288000
+ ~~~
    + CategoryInfo          : ObjectNotFound: (git:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

---

When I do GIT_TRACE=1 GIT_CURL_VERBOSE=1 git clone <repository-url> it does this:
GIT_TRACE=1 : The term 'GIT_TRACE=1' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ GIT_TRACE=1 GIT_CURL_VERBOSE=1 git clone https://github.com/uhazgotme ...
+ ~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (GIT_TRACE=1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

---

If you are not at a loss yet, is there anything else I can try you can think of? Thank you
